### ISSUE TITLE
Configure SMS Service Center (SMSC)

### ISSUE THREAD

**Issue by Hiestaa** [SMSSecure/SMSSecure#190]
Hello guys. Sorry if it's not the right place to ask, couldn't find my answer anywhere else.

I'm looking for a way to configure the SMS Service center ([SMSC](https://en.wikipedia.org/wiki/Short_message_service_center)), which seems to not be configured properly as the app is not able to send any SMS (although it can receive them, and the default SMS application is perfectly able to send). 

Any idea? Did you hide this option on purpose? 

Thanks!

**Comment by BastienLQ** [comment]
Hi,

What's your device?
What Android version is it running?
What version and build ID of SMSSecure do you have?
Can you submit a log when you try to receive/send a SMS?

Thanks!

**Comment by Hiestaa** [comment]
Hi,

My device is a Wiko Cink Peax 2. Notice that it does support double sims at the same time and that I recently changed the carrier (keeping the same phone number). When the new sim has been activated, I had both (old and new one) in the phone. It was working fine before that, it stopped working since then. I very strongly suspect the SMSC to not be properly configured, or still be configured to match the one of the previous carrier.
Android is in version 4.1.2 (build number ALPS.JB.SP.V1)
SMS Secure Version 0.10.1, Build ID: 16e0618.

I'll provide you with the log as soon as I got it (the message is still pending after 25min). Note that I do receive the SMS properly and again, this happens with SMSSecure _only_, not the default sms application.

Thank you very much for your help !

**Comment by Hiestaa** [comment]
Hello,

I managed to fix the problem. It was caused by the double-sim configuration. I had:
* Slot SIM 1: old carrier, the sim was just deactivated.
* Slot SIM 2: new carrier, the sim was just activated.

By putting the new carrier's SIM into slot 1, I am now able to send SMS properly. I guess the application is not designed to manage double SIM and it was still trying to send the message via the deactivated sim in slot 1.
